doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -465,11 +465,11 @@ See the following instructions for programming the images separately:
 
       2. Navigate to the build folder of the application sample and enter the following command to erase the flash memory of the application core and program the application sample::
```

Comment:
This one says "navigate to the build folder", in which case it makes sense to say zephyr/zephyr.hex